ut/client_ut.cpp

```diff
@@ -1488,3 +1488,45 @@ TEST(SimpleClientTest, issue_335_reconnects_count) {
             << "\tThere was no attempt to connect to endpoint " << endpoint;
     }
 }
+
+TEST_P(ClientCase, QueryParameters) {
+    const std::string table_name = "test_clickhouse_cpp_query_parameter";
+    client_->Execute("CREATE TEMPORARY TABLE IF NOT EXISTS " + table_name + " (id UInt64, name String)");
+    {
+        Query query("insert into " + table_name + " values ({id: UInt64}, {name: String})");
+
+        query.SetParam("id", "1").SetParam("name", "NAME");
+        client_->Execute(query);
+
+        query.SetParam("id", "123").SetParam("name", "FromParam");
+        client_->Execute(query);
+
+        query.SetParam("id", "333")
+            .SetParam("name", std::string("A\000A\001A\002A\003A\004A\005A\006A\007A\010A\011A\012A\013A\014A\015A\016A\017A\020A\021A\022A"
+                                          "\023A\024A\025A\026A\027A\030A\031A\032A\033A\034"
+                                          "A\035A\036A\037A",
+                                          65));
```

Comment:
Again, hardcoded value where you've been using some sort of a constant
```
    const char big_string_value[] = "A\000A\001A\002A\003A\004A\005A\006A\007A\010A\011A\012A\013A\014A\015A\016A\017A\020A\021A\022A"
                                          "\023A\024A\025A\026A\027A\030A\031A\032A\033A\034"
                                          "A\035A\036A\037A";
```

and then do `sizeof(big_string_value) - 1`, accounting for a zero terminator.